src/Tizen.NUI/src/public/WebView/WebBackForwardList.cs

```diff
@@ -75,7 +75,7 @@ public string OriginalUrl
     /// <summary>
     /// WebBackForwardSubList is a class for back-forward copied list item of web view.
```

Comment:
please rearrange this sentence, it might improve readability if you keep the name of item class literal, maybe sth like "WebBackForwardSubList contains a copy of list of WebBackForwardListItem", or maybe a copy of fragment, but then - why a saying it's a copy.

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520